encrypted_search/index.py

```diff
@@ -0,0 +1,63 @@
+from typing import List, Set, Tuple
+
+from utils.normalizer import normalize
+from utils.types import corpus_type, event_type, index_type
+
+
+class EncryptedIndex:

```

Comment:
Ok this is a totally minor thing, but: What would you think about calling the class something like `EncryptableIndex`?  Since it's not really encrypted, but it's made so that it easily can be.

Reply:
I think it should be `Encrypted` keeping with the idea of structural encryption?